packages/grafana-ui/src/components/Layout/Box/Box.tsx

```diff
@@ -290,6 +301,12 @@ const getStyles = (
       getResponsiveStyle(theme, gap, (val) => ({
         gap: theme.spacing(val),
       })),
+      getResponsiveStyle(theme, minWidth, (val) => ({
+        minWidth: val,
```

Comment:
Wonder if this should be a value for the `spacing` function instead like it's done in other components 🤔

Reply:
Yeah, the rationale for using it is that if in the future we decide to switch to using sizing tokens, it'll be easier if the props are already aligned with the spacing function.